### PR TITLE
[Connector-V2-e2e] Add file connector e2e test cases (#2171)

### PR DESCRIPTION
**Pull request by TyrantLucifer**
## Purpose of this pull request

#2171 

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by Hisoka-X** [comment]
The ci should passed first.

**Comment by TyrantLucifer** [comment]
> The ci should passed first.

Because the flink docker name had not been changed, so test case will has some problems, let's waiting the pr #2297 merged and restart CI again.

**Comment by Hisoka-X** [comment]
> > The ci should passed first.
> 
> Because the flink docker name had not been changed, so test case will has some problems, let's waiting the pr #2297 merged and restart CI again.

Merged, please merge from dev again.

**Comment by TyrantLucifer** [comment]
> > > The ci should passed first.
> > 
> > 
> > Because the flink docker name had not been changed, so test case will has some problems, let's waiting the pr #2297 merged and restart CI again.
> 
> Merged, please merge from dev again.

Done, let's waiting CI.

File: .github/workflows/documents.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+jobs:
+  build:
+    name: Build website
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout PR
+        uses: actions/checkout@v3
+        with:
```
Comment (CalvinKirs) [inline .github/workflows/documents.yml]: You've introduced other unrelated files, I'm guessing because of the wrong way of merging, please revive these changes

Comment (TyrantLucifer) [reply .github/workflows/documents.yml]: Fixed, let's waiting CI.

**Comment by TyrantLucifer** [comment]
@Hisoka-X @CalvinKirs  CI has done, please review.

**Review by Hisoka-X — APPROVED** [review]
